[docs] add note on having multiple apps for fcm (#18526)
expo/expo: docs/pages/push-notifications/using-fcm.md

```javascript
{
  ...
  "android": {
    "googleServicesFile": "./google-services.json",
    ...
  }
}
```

5. Confirm that your API key in `google-services.json` has the correct "API restrictions" in the [Google Cloud Platform API Credentials console](https://console.cloud.google.com/apis/credentials). For push notifications to work correctly, Firebase requires the API key to either be unrestricted (the key can call any API), or have access to both `Firebase Cloud Messaging API` and `Firebase Installations API`. The API key can be found under the `client.api_key.current_key` field in `google-services.json`.

```javascript
{
  ...
  "client": [
    {
      "api_key": [
        {
          "current_key" "<your Google Cloud Platform API key>",
        }
      ],
    }
  ]
}
```

> **Note:** Firebase will create an API key in the Google Cloud Platform console with a name like `Android key (auto created by Firebase)`. **This is not always the same key as the one found in `google-services.json`. Always confirm your key and associated restrictions in the Google Cloud Platform console.**

> > Firebase projects with multiple Android Apps might contain duplicated data under the `client` array in the download `google-services.json`, this may cause issues when the App is fetching the push notification token. **Be sure to only have one client object with the correct keys and metadata in `google-services.json`**.

Reviewer: ```suggestion
> Firebase projects with multiple Android apps might contain duplicated data under the `client` array in the downloaded `google-services.json`. This can cause issues when the app is fetching the push notification token. **Make sure to only have one client object with the correct keys and metadata in `google-services.json`**.
```

Small changes to follow our [writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md) and updating the overall verbiage.
